### PR TITLE
Stage 091 readme

### PR DESCRIPTION
**Pull request by Evildoor**
- [x] Clean, update and clarify the README
- [x] Wait for #199, then merge `master` here to add the samples' description

### CODE REVIEW

File: Utils/Dataflow/091_datasetsRucio/README [hunk 1]
```diff
@@ -49,3 +49,15 @@ Being able to do so requires:
...
 https://twiki.ific.uv.es/twiki/bin/view/Atlas/AtlasDataProcessingAtIFIC
+
+3. Running the stage
+--------------------
+Stage can be run as following:
+
+  ./datasets_processing.py filename.ndjson
+
+which will process the file in output mode and produce the result
+filename.json in the directory of the file. For more information about
```
[the remark this answers is not in the document]
Comment (Evildoor) [reply Utils/Dataflow/091_datasetsRucio/README]: I believe that the current version is better: the new version `... process the file ... containing the input file.` can make an incorrect impression that `file` and `input file` are different ones.

File: Utils/Dataflow/091_datasetsRucio/README [hunk 2]
```diff
@@ -14,8 +14,38 @@ in one of the two ways:
...
-  (don`t forget to fix USERNAME and, again, path to cert files)
+* proper configuration file (see below)
+* proper memberships and grid certificate (see below)
+
+Rucio requires authentication, which is performed by supplying a number of
+things, such as certificate or user name. In case of this stage, the
+authentication is performed as following:
+
+1) The stage sets up the environment variable VIRTUAL_ENV, its value will be
+   "~/.rucio" (if it exists), or "./.rucio" otherwise.
```
[the remark this answers is not in the document]
Comment (Evildoor) [reply Utils/Dataflow/091_datasetsRucio/README]: Thanks for pointing this out. Done.

File: Utils/Dataflow/091_datasetsRucio/README (hunk 1 above)
[the remark this answers is not in the document]
Comment (mgolosova) [reply Utils/Dataflow/091_datasetsRucio/README]: My point was that "the file" may also be interpreted as "filename.json", which makes no sence. And as "filename.json" is the last file mentioned before "the file", it is very likely to happen.

Maybe this way:
```
Stage can be run as follows:

  ./datasets_processing.py filename.ndjson

This command will process the input file (filename.ndjson) in output mode
and save the results in new file (filename.json). The output file will
be created in the directory containing the input file.
```

Or even use `path/to/input/file.ndjson` and `path/to/input/file.json` to make things more illustrative.

[the remark this answers is not in the document]
Comment (Evildoor) [reply Utils/Dataflow/091_datasetsRucio/README]: Implemented the latter.

**Pull request merged**